alhimick-prog/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+INPUT_PROMPT = 'Enter city name:'
+ERROR_BLANK_STRING = 'City name can not be blank!'
+ERROR_FILE_READ_FAIL = 'Error: data file not found'
+ERROR_MOUNT_NUMBER = 'Error: something went wrong when counting months.'
+MEAN = 'Rainfall mean:'
+VARIANCE = 'Rainfall variance:'
+EXIT_STRING = 'exit!'
+TOWN_ERROR = -1
```

Comment:
@alhimick-prog what do you think about `TOWN_ERROR_NUMBER`?

Reply:
@lincAlinc in the condition of the problem on the codewars:
"if functions `mean` or `variance` have as parameter `town` a city which has no records return `-1` or `-1.0` (depending on the language)"
In this constant I store the value that needs to be returned in accordance with the condition of the task.
Maybe it makes sense to rename to `MESSAGE_ERROR_TOWN_SEARCH`?

Reply:
@alhimick-prog -1 - it's number, you store this as integer. I think `TOWN_SEARCH_ERROR_NUMBER`

Reply:
how about `TOWN_SEARCH_ERROR_CODE` ? 😄